marketplace/connect/client.py

```diff
@@ -0,0 +1,77 @@
+import json
+import requests
+
+from django.conf import settings
+
+
+class ConnectAuth:
+    def get_auth_token(self) -> str:
+        request = requests.post(
+            url=settings.OIDC_OP_TOKEN_ENDPOINT,
+            data={
+                "client_id": settings.OIDC_RP_CLIENT_ID,
+                "client_secret": settings.OIDC_RP_CLIENT_SECRET,
+                "grant_type": "client_credentials",
+            },
+        )
+        token = request.json().get("access_token")
+        return f"Bearer {token}"
+
+    def auth_header(self, headers: dict = {}) -> dict:
```

Comment:
Can you explain me better why you need this `headers` parameter, please?

Reply:
I think in the future some endpoint can need a header parameter, so instead of just return a header with token, this method append the token to a existent header.